sense/loading.py

```diff
@@ -0,0 +1,183 @@
+        self.version = version
+        self.feature_converters = feature_converters
+
+    def get_path_weights(self):
+        model_weights = MODELS[self.model_name][self.version]
+        return {name: model_weights[name] for name in ['backbone'] + self.feature_converters}
+
+
+def get_relevant_weights(model_config_list: List[ModelConfig], requested_model_name=None,
+                         requested_version=None) -> Optional[Tuple[ModelConfig, dict]]:
+    """
+    Returns the model weights for the appropriate backbone and classifier head based on
+    a list of compatible model configs. The first available config is returned.
+
+    :param model_config_list:
+        List of compatible model configurations
+    :param requested_model_name:
+        Name of a specific model to use (i.e. StridedInflatedEfficientNet or StridedInflatedMobileNetV2)
+    :param requested_version:
+        Version of the model to use (i.e. pro or lite)
+    :return:
+        First available model config and dictionary of model weights
+    """
+
+    # Filter out model configurations that don't match requested name and version
+    if requested_model_name:
+        model_config_list = [config for config in model_config_list
+                             if config.model_name == requested_model_name]
+    if requested_version:
+        model_config_list = [config for config in model_config_list
+                             if config.version == requested_version]
+
+    # Check if not empty
+    if not model_config_list:
+        raise Exception(f'Could not find a model configuration matching requested parameters:\n'
+                        f'\tmodel_name={requested_model_name}\n'
+                        f'\tversion={requested_version}')
+
+    for model_config in model_config_list:
+        path_weights = model_config.get_path_weights()
+        path_weights_string = json.dumps(path_weights, indent=4, sort_keys=True)  # used in prints
+
+        files_exist = all(os.path.exists(prepend_resources_path(path)) for path in path_weights.values())
+        if files_exist or running_on_travis():
+            print(f'Weights found:\n{path_weights_string}')
+            weights = {}
+            for name, path in path_weights.items():
+                load_fn = load_backbone_weights if name == 'backbone' else load_weights_from_resources
+                weights[name] = load_fn(path)
+
+            return model_config, weights
+        else:
+            print(f'Could not find at least one of the following files:\n{path_weights_string}')
+
+    raise Exception('ERROR - Weights file missing. To download, please go to '
+                    'https://20bn.com/licensing/sdk/evaluation and follow the '
+                    'instructions.')
+
+
+def prepend_resources_path(checkpoint_path):
+    """
+    Prepend the absolute resources path to the provided path.
+    """
+    return os.path.join(RESOURCES_DIR, checkpoint_path.split(f'resources{os.sep}')[-1])
```

Comment:
The split on `resources/` should not necessary, now that  the paths no longer contain it.